3.analysis/scripts/nbconverted/3.t_tests.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import sys
+import pandas as pd
+sys.path.insert(0, ".././0.data-download/scripts/")
+from data_loader import load_data
+from scipy.stats import ttest_ind
+
+
+# In[2]:
+
+
+latent_df = pd.read_csv("../2.train-VAE/results/latent_df.csv")
+metadata_df = pd.read_csv(".././0.data-download/data/metadata_df.csv")
+data_dir = "../0.data-download/data/"
+model_df, dependency_df = load_data(data_dir, adult_or_pediatric="all")
+
+
+# In[3]:
+
+
+# Creating categorized lists of sample IDs used in BVAE training
+# note that 10 of the 912 used samples have Unknown Sex
+
+ped_ids = []
+adult_ids = []
+male_ids = []
+female_ids = []
+ped_male_ids = []
+adult_male_ids = []
+ped_female_ids = []
+adult_female_ids = []
+
+for index, row in metadata_df.iterrows():
```

Comment:
You're going to love [`pandas.DataFrame.query()`](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.query.html) 😁

Reply:
Working on implementing now